admin_manual/exapps_management/AppAPIAndExternalApps.rst

```diff
@@ -0,0 +1,70 @@
+
+Most of our :doc:`../ai/index` (AI) apps are developed as ExApps and thus may require some preparation of your Nextcloud instance before you can install them.
+
+Installing AppAPI
+-----------------
+
+All ExApps require the `AppAPI <https://apps.nextcloud.com/apps/app_api>`_ Nextcloud app as a dependency.
+As of Nextcloud 30, AppAPI is automatically installed by default.
+If AppAPI is not installed, you can still install it by simply navigating to the Apps management page in your Nextcloud instance and search for AppAPI from the Tools category.
+
+Setup deploy daemon
+-------------------
+
+A Deploy Daemon is a way for Nextcloud to install and communicate with and control ExApps.
+
+.. note::
+	If you are using Nextcloud AIO with the "Docker Socket Proxy" container enabled, a Deploy Daemon will be automatically created and configured to work out-of-the-box.
+	Otherwise, follow the steps below to set up a Deploy Daemon from the AppAPI admin settings.
+
+1. Setup a Docker container called `docker-socket-proxy <https://github.com/nextcloud/docker-socket-proxy#readme>`_ that proxies access to Docker for your Nextcloud instance.
+2. Go to the AppAPI admin settings.
+3. Click on the "Register Daemon" button.
+4. Fill in the required fields:
+	- ``Name``: unique name of the Deploy daemon
+	- ``Display name``: the name that will be displayed in the UI
+	- ``Deployment method``: by default you will need to choose ``docker_install``, ``manual_install`` is for development or custom use case of manual ExApp installation
+	- ``Daemon Host``: hostname/IP address + port of the Deploy daemon
+	- ``Nextcloud URL``: autofilled with current domain, you might need to change the protocol to http/https depending on your setup
+	- ``Set as default daemon``: check if you want set new Deploy daemon as default
+	- ``Enable https``: check if your Deploy daemon (Docker Socket Proxy) is configured with TLS
+	- Deploy Config:
+		- ``Network``: Docker network name, depends on your networking setup, enforces to "host" if "Enable https" is checked
+		- ``HaProxy password``: password for Docker Socket Proxy, if it is configured with TLS
+		- ``Compute Device``: CPU, CUDA or ROCm, depending on your hardware config on Deploy daemon host machine
+		- ``Add additional option`` (see :ref:`additional_options_list`): setup additional KEY + VALUE deploy config options
+5. Click "Check connection" to verify that the configuration is correct.
+6. Click "Register" to save the Deploy Daemon configuration.
+
+Deployment configuration examples can be found :ref:`here <deploy-configs>`.
+
+Installing ExApps
+-----------------
+
+You can now install ExApps from the Nextcloud App Store by clicking "Install" on the respective app in the Apps page.
+If successful, the ExApp will be displayed under the "Your apps" list.
+
+.. image:: ./img/exapp_list_example.png
+
+FAQ
```

Comment:
part not related to AppAPI?

Reply:
I think there should be more of a focus on the ExApps themselves rather than AppAPI, or am I reading this wrong?